src/czbenchmarks/tasks/single_cell/k562_perturbation.py

```diff
@@ -0,0 +1,288 @@
+from typing import List, Dict
+import pandas as pd
+import numpy as np
+import logging
+from ..task import Task, TaskInput, TaskOutput
+from ...tasks.types import CellRepresentation
+from ...metrics import metrics_registry
+from ...metrics.types import MetricResult, MetricType
+from ...constants import RANDOM_SEED
+
+from tqdm import tqdm
+
+logger = logging.getLogger(__name__)
+
+
+def binarize_values(y_true, y_pred):
```

Comment:
This could go into a shared utils folder if it is likely to be used in other tasks.